src/app/components/hmcts-global-header/hmcts-global-header.component.html

```diff
@@ -69,11 +69,11 @@
         <ul class="hmcts-primary-navigation__list">
           <li class="hmcts-primary-navigation__item" *ngFor="let item of (rightItems | async)">
             <a class="hmcts-primary-navigation__link transparent-background" [ngClass]="item.ngClass" (click)="onEmitSubMenu(item)"
-            [attr.aria-current]="item.active ? true : false" [routerLink]="item.href" *ngIf="showFindCase && item.ngClass === 'hmcts-search-toggle__button'">{{item.text}}</a>
+            [attr.aria-current]="item.active ? true : null" [routerLink]="item.href" *ngIf="item.ngClass === 'hmcts-search-toggle__button'">{{item.text}}</a>
```

Comment:
Not sure if checking for a class to toggle the item is the best way.

Reply:
Thank you @ronaldmansveld - Changing it to false will display the active underline for all the menu items. It was null before, and @Daniel-Lam changed it to false as per review comments from Paul.
We looked at it and decided to revert to the previous value which was null.